client/src/core/client/stream/tabs/Notifications/polling/useLiveNotificationsPolling.tsx

```diff
@@ -43,7 +58,7 @@ export default function useLiveNotificationsPolling(userID?: string) {
     `);
 
   const updater = useCallback(async () => {
-    if (!userID) {
+    if (!userID || !settings?.inPageNotifications?.enabled) {
```

Comment:
Nice! Should we also check that in-page notifications is enabled in the viewer's preferences? (`viewer?.inPageNotifications?.enabled`)